### ISSUE TITLE
The column header buttons of the "Results" tab appeared after running the query are not accessible with keyboard. : A11y_SQL Azure Data Studio_Query_Keyboard

### ISSUE THREAD

**Issue by Mana-swi** [microsoft/azuredatastudio#26395]
### Tags:
#A11yTCS;#A11yWCAG;#WCAG2.1.1;#Keyboard;#A11ySev2;#Win32;#DesktopApp;#BM_SQL Azure Data Studio_Win32_Sep25;#SQL Azure Data Studio;
**"[[Check out Accessibility Insights!]](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)) - Identify accessibility bugs before check-in and make bug fixing faster and easier."**
### Environment Details:
Application Name: SQL Azure Data Studio
Version: 1.52.0 (system setup)
VS Code: 1.82.0
Electron: 30.5.1
Chromium: 124.0.6367.243
Node.js: 20.16.0
V8: 12.4.254.20-electron.0
OS: Windows_NT x64 10.0.22621
### Repro Steps:

1. Launch Azure Data Studio.
2. Navigate to File > New Query
3. Connect to a Server in the Query Editor and choose the credential, and optionally the database(DW Configuration) to connect to
4. Type a SQL query statement that produces a result set (SELECT * FROM dbo.compute_node)
5. enter Run
6. enter on the Results tab. 
7. Tab till results table.
8. Observe that The column header buttons of the "Results" tab appeared after running the query are not accessible with keyboard

### Actual:
The column header buttons of the "Results" tab appeared after running the query are not accessible with keyboard

### Expected:
The column header buttons of the "Results" tab appeared after running the query should be accessible with keyboard

### User Impact:
Keyboard users will not be able to interact or apply any filters or change the sorting orders as the column headers buttons are not accessible with keyboard.
### Recommendations:
 [MAS Problems and Suggested Fixes](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2fmicrosoft.sharepoint.com%2fsites%2faccessibility%2fLists%2fMASProblemsAndSuggestedFixes%2fReviewed%2520Items.aspx&data=04%7c01%7cv-prajadhav%40microsoft.com%7cf642cb2fc53d4c4be38408d9d97cf216%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637779952946112692%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c3000&sdata=avswAOFlXEiWy%2fRT%2f%2fPGnurQzMGOUdSW7Z5Xp8ZioiQ%3D&reserved=0)
### MAS Reference:

- [MAS 2.1.1 – Keyboard](https://aka.ms/MAS2.1.1)
- [MAS 2.1.1: Keyboard Guide](https://microsoft.sharepoint.com/sites/accessibility/SitePages/MAS-2.1.1-Keyboard-Guide.aspx)
